src/core/osmLayer.js

```diff
@@ -43,6 +43,7 @@ geo.osmLayer = function (arg) {
     m_numberOfCachedTiles = 0,
     m_tileCacheSize = 100,
     m_baseUrl = 'http://tile.openstreetmap.org/',
+    m_subdomains = 'abc',
```

Comment:
@jbeezley yes, for otile, we have 1,2,3,4, and for some others a,b,c (I know that) sicne we used default as tile.open***.org, I set the default to abc. Are you suggesting we leave it empty?